Module-generation/frontmatter.py

```diff
@@ -0,0 +1,75 @@
+import os
+import json
+
+DIR = "./meta-data/"
+DST = "./modules/"
+os.mkdir(DST)
+
+for folder in os.listdir(DIR):
+    moduleFile = DIR+folder+"/module.txt"
+    # check for Module.txt to get infomation about module
+    if(os.path.isfile(moduleFile)):
+        getModuledata = open(moduleFile, "r")
+        readModuleData = getModuledata.read()
+        parseData = json.loads(readModuleData)
+        moduleName = parseData['id']
+        moduleDescription = parseData['description']
+        moduleLogo = "defaultBanner.png"
+        moduleReadme = DIR+folder+"/README.md"
+
+        os.mkdir(DST+moduleName)
+        IndexMd = open(DST+moduleName+"/index.md", "a+")
```

Comment:
Do you ever close this?
You could consider using `with open() as indexMd:`, thus as soon as you exit that scope, the file will be closed.

Reply:
done